app/uk/gov/hmrc/pillar2stubs/controllers/FinancialDataController.scala

```diff
@@ -16,7 +16,7 @@
 
 package uk.gov.hmrc.pillar2stubs.controllers
 
-import play.api.libs.json.{JsArray, JsObject, JsValue, Json}
+import play.api.libs.json._
```

Comment:
Can you revert back the scalafix rule for imports back to 5 so we don't have to do that many wildcard imports?